variables.tf

```diff
@@ -188,6 +188,16 @@ variable "versioning_status" {
   }
 }
 
+variable "retention_status" {
```

Comment:
Hm. If I saw `retention_status = "Disabled"` in the code without having read the docs for the module, I would think that means "retention is disabled". However, it actually means logs _will_ be retained, because this setting disables the rule that expires the logs. Is that correct? If so, could we rename the var?

Reply:
Good suggestion! made an update to rename the variable `s3_log_bucket_lifecycle_rule`